src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -64,17 +64,19 @@
 
   sections: [
     {
-      name: 'basicInformation',
-      permissionRequired: true
-    },
-    {
-      name: 'requestDetails'
+      name: 'requestDetails',
+      title: 'Approval Request Information',
+      order: 0
     },
     {
-      name: 'supportDetails'
+      name: 'complianceInformation',
+      title: 'Compliance',
+      order: 1
     },
     {
-      name: '_defaultSection',
+      name: 'systemInformation',
```

Comment:
'_defaultSection' used by system as section for properties without a section specified btw. So from what I see every property has a section now?

Reply:
Ya they do, I could add it back in incase we add other properties that aren't section specified.

Reply:
re: _defaultSection - going forward we should be more conscious of new properties.